src/metrics/reporters/datadog.js

```diff
@@ -95,18 +95,19 @@ class DatadogReporter extends BaseReporter {
 
 		if (series.length == 0) return;
 
-		return fetch(
-			`${this.opts.baseUrl}${this.opts.apiVersion}${this.opts.path}?api_key=${this.opts.apiKey}`,
-			{
-				method: "post",
-				body: JSON.stringify({ series }),
-				headers: {
-					"Content-Type": "application/json"
-				}
+		return fetch(`${this.opts.baseUrl}${this.opts.apiVersion}${this.opts.path}`, {
```

Comment:
If the modified reporter only support v2, we should lock the apiVersion in the URL.

Reply:
Not sure, up to you. Will lock it.